fabric-datafixer-v1/src/main/java/net/fabricmc/fabric/datafixer/mixin/MixinPlayerEntity.java

```diff
@@ -0,0 +1,11 @@
+package net.fabricmc.fabric.datafixer.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import net.minecraft.entity.player.PlayerEntity;
+
+@Mixin(PlayerEntity.class)
+public class MixinPlayerEntity {
+    // Add Mod DataVersions to writeCustomDataToTag
```

Comment:
this sounds too expensive and will just blow up tags. can we just write ones we need?

Reply:
If you are asking, we only write the section for datafixers which have been registered. So if only two mods register fixers then we only add two entries to DataVersion list.

If we are concerned about larger than expected CompoundTags then that is a concern that will require more discussion.